tests/foreman/ui/test_ansible.py

```diff
@@ -556,6 +556,87 @@ def test_positive_non_admin_user_access_with_usergroup(
             ansible_roles_table = session.host_new.get_ansible_roles(target_sat.hostname)
             assert ansible_roles_table[0]['Name'] == SELECTED_ROLE
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list('8')
+    def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
+        self,
+        rhel_contenthost,
+        module_target_sat,
+        module_org,
+        module_location,
+        module_activation_key,
+    ):
+        """Check that Ansible tasks that make changes on a host show as notice in the config report and
+        failed Ansible tasks show as errors in the config report
+
+        :id: 286048f8-0f4f-4a3c-b5c7-fe9c7af8a780
+
+        :steps:
+            1. Import Ansible Roles
+            2. Assign Ansible roles to a host
+            3. Run Ansible Roles on a host
+            4. Check Config Report
+
+        :expectedresults: Verify that any tasks that make changes on the host are listed as notice in the config reportCaseLevel and
+            Verify that any task failures are listed as errors in the config reportCaseLevel
+        """
+        module_target_sat.cli.Capsule.update(
+            {
+                'name': module_target_sat.hostname,
+                'location-ids': module_location.id,
+                'organization-ids': module_org.id,
+            }
+        )
+        SELECTED_ROLE = 'theforeman.foreman_scap_client'
+        id = module_target_sat.nailgun_smart_proxy.id
+        module_target_sat.api.AnsibleRoles().sync(
+            data={'proxy_id': id, 'role_names': SELECTED_ROLE}
+        )
```

Comment:
```suggestion
            SELECTED_ROLE = 'theforeman.foreman_scap_client'
            nc = module_target_sat.nailgun_smart_proxy
            nc.location = [module_location]
            nc.organization = [module_org]
             nc.update(['organization', 'location'])
            module_target_sat.api.AnsibleRoles().sync(data={'proxy_id': nc.id, 'role_names': SELECTED_ROLE})
```

Reply:
Added `nailgun_smart_proxy` and removed suggested one